lib/lf/fe/scalar_reference_finite_element.h

```diff
@@ -0,0 +1,357 @@
+#ifndef LF_FE_SCALAR_REFERENCE_FINITE_ELEMENT_H_
+#define LF_FE_SCALAR_REFERENCE_FINITE_ELEMENT_H_
+
+/**
+ * @file
+ * @brief Data structure representing simple finite elements
+ * @author Ralf Hiptmair
+ * @date October 2018
+ * @copyright MIT License
+ */
+
```

Comment:
Here includes are missing. My IDE complains that the namespace lf::assemble is not defined.